v20.2/show-backup.md

```diff
@@ -136,7 +136,7 @@ Field | Description
 
```

Comment:
Parameter list here still shows kv_option_list

Reply:
> Parameter list here still shows kv_option_list

Hey Liv, so I think what is happening here is that typically the options are a key and value and so in the synopsis diagrams this is more often than not noted as `kv_option_list`; hence that being pulled down into the parameter list, which then links to the possible options for that parameter. I need to check on what should be the default here since these are all noted differently between backup + restore vs. import vs. show backup. For now, I'll leave as is as the page refers to each instance of this in the same way, but I'll create an issue to investigate this!
Thank you for highlighting this 👍